### PR TITLE
Skip non-supported mimes

### PR DESCRIPTION
**Pull request by VicDeo**
Do not start session for unknown filetypes

### CODE REVIEW

File: lib/filter.php [hunk 1]
```diff
@@ -57,9 +57,13 @@ public static function write($content, $mimetype){
 		return $data;
 	 }
 	 
-	 public static function getAll(){
+	public static function getAll(){
 		 return array_keys(self::$filters);
 	 }
 	 
+	public static function isSupportedMimetype($mimetype){
```
Comment (LukasReschke) [inline lib/filter.php]: PHPDoc would be awesome

File: lib/filter.php [hunk 2]
```diff
@@ -55,11 +55,20 @@ public static function write($content, $mimetype){
...
+	public static function getAll(){
 		 return array_keys(self::$filters);
-	 }
+	}
 	 
- }
+	/**
+	 * Checks if mimetype is supported by the app
+	 * @param string $mimetype - checked mimetype
+	 * @return array
```
Comment (LukasReschke) [inline lib/filter.php]: bool :)

Comment (VicDeo) [inline lib/filter.php]: @LukasReschke oops, fixed :)

**Comment by LukasReschke** [comment]
Patch is confirmed. Would love to have this in stable7 and stable6 as well if possible.

@karlitschek

**Comment by LukasReschke** [comment]
Btw. thanks for the proactive patching. Good stuff :+1:

**Comment by karlitschek** [comment]
please backport

**Comment by LukasReschke** [comment]
Let's merge this then.

**Comment by LukasReschke** [comment]
@VicDeo Any chance you can create backports for that one? THX!

**Pull request merged**